lazy_tensor_core/lazy_bench.py

```diff
@@ -56,7 +56,7 @@
 current_name = ""
 current_device = ""
 
-@functools.cache
+@functools.lru_cache
```

Comment:
now I'm wondering why I used cache in the first place. Do you know what the difference is, and did you confirm the csv file still has the correct data before/after the change?

Reply:
No, I didn't... I was trying to move fast and rely on the CI.

Reply:
ok. well if it works then I am ok with the change. ill stamp to unblock but i'm assuming you'll make sure it's working before land.

Reply:
Yea, it should work as suggested here: https://docs.python.org/3/library/functools.html#functools.cache. But I will verify that before landing.